docs/modules/ROOT/partials/getting-started/con-registry-serdes-avro.adoc

```diff
@@ -4,43 +4,41 @@
 [id='registry-serdes-types-avro-{context}']
 = Configure Avro SerDe with {registry}
 
-{registry} provides Kafka client serializer and deserializer classes for Apache Avro to make using Avro as
-easy as possible:
-
-* `io.apicurio.registry.utils.serde.AvroKafkaSerializer`
-* `io.apicurio.registry.utils.serde.AvroKafkaDeserializer`
+[role="_abstract"]
+{registry} provides the following Kafka client serializer and deserializer classes for Apache Avro:
 
+* `io.apicurio.registry.serde.avro.AvroKafkaSerializer`
+* `io.apicurio.registry.serde.avro.AvroKafkaDeserializer`
 
 .Configure the Avro serializer
 
-You can configure the Avro serializer class in the following ways:
+You can configure the Avro serializer class with the following:
 
-* {registry} location as a URL
-* Artifact ID strategy 
+* {registry} URL
+* Artifact resolver strategy 
 * Global ID strategy 
 * Global ID location
 * Global ID handler
```

Comment:
There is no global id handler nor global id location concept. At least not with "Global Id" name :) . Depending on the location of the artifact id info ( headers or message payload) you can configure an Id Handler or a headers handler.

Reply:
OK, renaming these options to more generic "ID location" and "ID encoding".